packages/customer/src/services/customer-module.ts

```diff
@@ -775,4 +782,30 @@ export default class CustomerModuleService implements ICustomerModuleService {
         )
       : void 0
   }
+
+  private async flush(context: Context) {
```

Comment:
thought: This could live as a MikroORM util no? It only depends on the context

Reply:
yes it could - let's leave it here until we meet the pattern in other places